cpp/inc/bond/protocol/fast_binary.h

```diff
@@ -373,7 +373,7 @@ class FastBinaryReader
         uint32_t size;
 
         ReadContainerBegin(size, element_type);
-        for (uint32_t i = 0; i < size; ++i)
+        for (int64_t i = 0; i < size; ++i)
```

Comment:
Don't the `SkipType(size_t)` functions also need checking?

Reply:
I was planning to fix that separately. This change does not actually add any runtime checks.